### DISCUSSION TITLE
ホーム画面で同じユーザーの記事が複数並んで表示される

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#281] [要望・アイデア・不具合]
現在のQiitaのホーム画面はレコメンドされて表示される仕様だったと記憶しています。

最近、タイトルの通りにホーム画面で同じユーザーの記事が複数表並んで示されることが多いと感じました。
参考までに今日、Qiitaを開いた時の画面をキャプチャしたので添付します。

![スクリーンショット 2022-08-24 9 06 39](https://user-images.githubusercontent.com/38848514/186325039-afcacc2e-9a31-4e93-ad45-bc2040b38033.png)
![スクリーンショット 2022-08-24 9 06 59](https://user-images.githubusercontent.com/38848514/186325046-49ea96d2-85bc-4307-aae6-d045caa717d1.png)

「もっと読む」を押して記事を読み込んでも、同じユーザーの記事が複数並んでいます。さすがに今日のホーム画面は開いた時にびっくりしたので、discussionさせていただきました。

感覚的に同じユーザーの記事が複数並んでいると、気味が悪く感じてしまうため、同じユーザーの記事は1記事に限定した方がUX的に良いのではないかと感じた次第です。（逆に自分の記事も表示されている可能性を考えると、気持ち悪いですよね。）

つきましては、ホーム画面について改善の検討をお願いします。

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご意見ありがとうございます🙏
こちら #244 でも同様のご意見をいただいており、我々も課題に感じております🙇
見せ方など模索中でして、今後のトップページのアップデートの際に検討していきたいと思っております。

**Reply by Brutus03** [reply]
@tanaka-asami415 
 #244 は2ヶ月前に起票されていますね。
見せ方を模索するのは良いと思いますが、同じユーザーの記事が複数並んで表示されるのだけでも、優先度上げて対応いただけると幸いです。

どうぞ、よろしくお願いします。

**Reply by Brutus03** [reply]
すぐには改善されないと思っているので、一ユーザーとしての意見を書かせていただきます。
相変わらず添付のような気味悪さよりは、昔のようなトレンドが表示されている方が個人的には好みでした。
#10 と同じ話ですが、見向きもしない今のホーム画面を見せ続けられるよりはトレンドの方が良いと思います。

![スクリーンショット 2022-09-03 19 18 13](https://user-images.githubusercontent.com/38848514/188804901-871c5a70-376c-4b97-b305-68fe1e672726.png)
![スクリーンショット 2022-09-03 19 18 20](https://user-images.githubusercontent.com/38848514/188804912-92ddef0b-7303-4af1-8b77-49420f4af299.png)

**Reply by tanaka-asami415** [reply]
@Brutus03 
ご不便をおかけし、申し訳ありません。こちら、まもなく着手予定です！
実装までには時間がかかってしまう可能性がありますが、もうしばらくお待ちいただけると幸いです🙇💦

**Comment by kato647** [comment]
@Brutus03 
大変お待たせいたしました :bow:
ホーム画面の記事の表示ロジックを改修しました。
ホーム画面をご確認いただければと思います！

**Reply by Brutus03** [reply]
@kato647 
ホーム画面確認しました。同じユーザーの記事が複数並んで表示されていないので、大丈夫だと思います！

なお、タイムラインの「すべて」については、同じ事象が発生していますが、私はほとんどタイムラインを見ることはないので気にしていません。

単純にQiitaにログインしている状態で、Qiitaを開くとデフォルトはホーム画面です。
その時に、同じユーザーの記事が複数並んで表示されることが気味が悪かったため、事象改善したのであればクローズで大丈夫です。
お忙しい中、ご対応ありがとうございました。
